server/utils/auth.ts

```diff
@@ -0,0 +1,41 @@
+// server/utils/auth.ts
+import { Lucia } from "lucia";
+import { Twitch } from "arctic";
+
+import { DrizzleMySQLAdapter } from "@lucia-auth/adapter-drizzle";
+import { sessionTable, userTable } from "./schema";
+import { drizzle } from "drizzle-orm/mysql2";
+import mysql from "mysql2";
+
+const connection = mysql.createConnection({
+	host: process.env.DB_HOST || "localhost",
+	user: process.env.DB_USER || "root",
+	password: process.env.DB_PASS || "",
+	database: process.env.DB_NAME || "greasy",
+});
```

Comment:
We should create the db connection in a dedicated file so we can access it everywhere